content/sensu-go/5.14/release-notes.md

```diff
@@ -40,6 +41,42 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
 
 ---
 
+## 5.14.0 release notes
+
+**October 3, 2019** &mdash; The latest release of Sensu Go, version 5.14.0, is now available for download. This release includes feature additions like two new configuration options for backends using embedded etcd and a new SemVer field in entity resources. In addition, this release includes bug fixes that affect round robin schedule recovery after quorum loss and enhanced TSL authentication support.
```

Comment:
"affect" or "enable"? I'm wondering if we can include stronger language around there being a fix.

Reply:
also, should it be TLS authentication?

Reply:
Yes to both, thank you!